CHANGELOG.md

```diff
@@ -1,20 +1,36 @@
 # Change Log
 All notable changes to this project will be documented in this file.
-This project adheres to the Node default version scheme, meaning It's safe to use with the caret ^ dependency definition.
-The standard for the caret can [be found here](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004).
-Breaking changes result in a different major. UI changes that might break customizations on top of the sdk, will be treated as breaking changes too.
+
+This change log file is based on best practices from [Keep a CHANGELOG](http://keepachangelog.com/).  
+This project adheres to the Node default version scheme, meaning it is safe to use with the caret ^ dependency definition. The standard for the caret can [be found here](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004).  
+This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 
 ## [next-version]
 
+## [1.0.0]
+
+We are bumping version to 1.0.0 because:
+* SDK has already been implemented in production integrations
+* changes introduced in this version are not backwards-compatible
+* we are following Semantic Versioning for consistency
```

Comment:
Point 1 is sensible. Point 2 & 3 are implicit - they are not the reason we are bumping to 1.0.0. I would remove them then add the main reason we are bumping to 1.0.0 which is that we now integrate with the Onfido API.